### PR TITLE
Reviung53 shield definition

### PR DESCRIPTION
**Pull request by kh411d**
## Board/Shield Check-list

- [x] This board/shield is tested working on real hardware
- [x] Definitions follow the general style of other shields/boards upstream ([Reference](https://zmk.dev/docs/development/new-shield))
- [x] `.zmk.yml` metadata file added
- [x] Proper Copyright + License headers added to applicable files (Generally, we stick to "The ZMK Contributors" for copyrights to help avoid churn when files get edited)
- [x] General consistent formatting of DeviceTree files
- [x] Keymaps do not use deprecated key defines (Check using the [upgrader tool](https://zmk.dev/docs/codes/keymap-upgrader))
- [x] `&pro_micro` used in favor of `&pro_micro_d/a` if applicable
- [n/a] If split, no name added for the right/peripheral half
- [x] Kconfig.defconfig file correctly wraps _all_ configuration in conditional on the shield symbol
- [x] `.conf` file has optional extra features commented out
- [x] Keyboard/PCB is part of a shipped group buy or is generally available in stock to purchase (OSH/personal projects without general availability should create a zmk-config repo instead)

### CODE REVIEW

File: app/boards/shields/reviung53/reviung53.keymap [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+#include <dt-bindings/zmk/rgb.h>
+
+/ {
+        conditional_layers {
+                compatible = "zmk,conditional-layers";
+                tri_layer {
+                if-layers = <1 2>;
+                then-layer = <3>;
+                };
+        };
```
Comment (caksoylar) [inline app/boards/shields/reviung53/reviung53.keymap:19]:
```suggestion
    conditional_layers {
        compatible = "zmk,conditional-layers";
        tri_layer {
            if-layers = <1 2>;
            then-layer = <3>;
        };
    };
```

File: app/boards/shields/reviung53/reviung53.keymap [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+// |          |          |    ESC   |          |          |           |        |        |        |       |      |   DEL   |
+// | TAB      |          |    UP    |          |          |           |        |        |        |       |      |  BSPC   |
+// | NAV      |   LEFT   |   DOWN   |   RIGHT  |          |           |  LEFT  |  DOWN  |  UP    | RIGHT |      |  ENTER  |
+// | SHIFT    |          |          |          |          |           |  HOME  |  END   |  PGUP  | PGDN  |     SHIFT      |
+// | CTRL     |   GUI    |  ALT     |            SPACE                |      SPACE      |  ALT   | GUI   |     CTRL       |
+//
+                        bindings = <
+                          &kp ESC   &none      &none     &none     &none      &none     &none      &none      &none     &kp DEL
+   &kp TAB      &none     &kp UP    &none      &none     &none     &none      &none     &none      &none      &none     &kp BSPC
+   &mo 4        &kp LEFT  &kp DOWN  &kp RIGHT  &none     &none     &kp LEFT   &kp DOWN  &kp UP     &kp RIGHT  &none     &kp RET
```
Comment (caksoylar) [inline app/boards/shields/reviung53/reviung53.keymap:103]:
```suggestion
   &trans       &kp LEFT  &kp DOWN  &kp RIGHT  &none     &none     &kp LEFT   &kp DOWN  &kp UP     &kp RIGHT  &none     &kp RET
```

File: app/boards/shields/reviung53/reviung53.zmk.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+file_format: "1"
+id: reviung53
+name: REVIUNG53
+type: shield
+url: https://github.com/gtips/reviung/tree/master/reviung53
+requires: [pro_micro]
+features:
+  - keys
```
Comment (caksoylar) [inline app/boards/shields/reviung53/reviung53.zmk.yml:8]:
```suggestion
  - keys
  - underglow
```

**Review by caksoylar — COMMENTED** [review]
In addition to the suggestions I noted below, can you update the copyright header dates on all files?

**Comment by kh411d** [comment]
Change addressed, copyright date updated, keymap indentation adjusted.

**Review by caksoylar — APPROVED** [review]

**Review by petejohanson — APPROVED** [review]
Thanks!

**Pull request merged**